packages/gasket-request/test/postbuild-replace.test.js

```diff
@@ -0,0 +1,111 @@
+import { execSync } from 'node:child_process';
```

Comment:
What is this file for?

Reply:
I added it as a test for the updates to postbuild:replace npm script in this package.